### PR TITLE
Vendor cleanup

### PR DESCRIPTION
**Pull request by vrothberg**
Some cleanups for the dependencies. I pinned branches to commits or releases (where possible) and added a `make vendor` target. We might consider integrating `make vendor` into Travis to make sure the vendor.conf is in sync with `./vendor`.

Cc @TomSweeneyRedHat @nalind @rhatdan

### CODE REVIEW

**Comment by vrothberg** [comment]
bot, retest this please

File: vendor.conf [hunk 1]
```diff
@@ -10,59 +10,55 @@ github.com/containers/storage v1.8
...
+golang.org/x/net 45ffb0cd1ba084b73e26dee67e667e1be5acce83 https://github.com/golang/net
+golang.org/x/sync 37e7f081c4d4c64e13b10787722085407fe5d15f https://github.com/golang/sync
+golang.org/x/sys 7fbe1cd0fcc20051e1fcb87fbabec4a1bacaaeba https://github.com/golang/sys
+golang.org/x/text e6919f6577db79269a6443b9dc46d18f2238fb5d https://github.com/golang/text
 gopkg.in/cheggaaa/pb.v1 v1.0.27
 gopkg.in/yaml.v2 v2.2.2
-k8s.io/apimachinery master
-k8s.io/client-go master
-k8s.io/kubernetes master
+k8s.io/client-go kubernetes-1.10.13-beta.0 https://github.com/kubernetes/client-go
```
Comment (TomSweeneyRedHat) [inline vendor.conf]: does this cover apimachinery?

Comment (vrothberg) [reply vendor.conf]: Is apimachinery needed? vndr stated it's not and a quick grep in the code didn't reveal a dependency.

File: vendor.conf [hunk 2]
```diff
@@ -10,59 +10,55 @@ github.com/containers/storage v1.8
...
-github.com/opencontainers/runc master
+github.com/opencontainers/runc v1.0.0-rc6
 github.com/opencontainers/runtime-spec v1.0.0
 github.com/opencontainers/runtime-tools v0.8.0
 github.com/opencontainers/selinux v1.1
 github.com/openshift/imagebuilder a4122153148e3b34161191f868565d8dffe65a69
 github.com/ostreedev/ostree-go 9ab99253d365aac3a330d1f7281cf29f3d22820b
-github.com/pborman/uuid master
-github.com/pkg/errors master
+github.com/pkg/errors v0.8.1
```
Comment (TomSweeneyRedHat) [inline vendor.conf]: were we not really using uuid?

Comment (vrothberg) [reply vendor.conf]: Nope

**Comment by TomSweeneyRedHat** [comment]
@vrothberg as you've looked through the code and didn't find those removed ones in use, then:
LGTM assuming happy tests

**Comment by vrothberg** [comment]
@TomSweeneyRedHat do you know if the Red Hat CI has some hiccups atm?

**Comment by TomSweeneyRedHat** [comment]
IDK at the moment, but it's been a 50/50 success rate lately.  I'd restart it after Travis finishes.

**Comment by vrothberg** [comment]
bot, retest this please

**Comment by vrothberg** [comment]
@TomSweeneyRedHat we're green :)

**Comment by vrothberg** [comment]
:-D well Travis seemed to have changed sides

**Comment by vrothberg** [comment]
Ah, gingko needs to come back. I'll whitelist it in make vendor.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**